assets/src/data/content/model/schema.ts

```diff
@@ -25,7 +49,7 @@ const tableCell = {
   isVoid: false,
   isBlock: true,
   isTopLevel: false,
-  validChildren: toObj(['p', 'img', 'youtube', 'audio', 'math', 'formula_inline', 'formula']),
+  validChildren: toObj(['p', 'math', 'formula_inline', 'formula', ...MediaElements]),
```

Comment:
Here's an example of the type of thing I was trying to clean up since 'video' should have been included.